phalcon/dispatcherinterface.zep

```diff
@@ -125,8 +123,6 @@ interface DispatcherInterface
 
 	/**
 	 * Forwards the execution flow to another controller/action
-	 *
-	 * @param array forward
 	 */
-	public function forward(forward);
+	public function forward(array forward);
```

Comment:
Yea i agree. Only one problem can be if someone has his own dispatcher classes. I would just keep old syntax, and mark todo to change it in 4.0.0 to array because i think there is really no point of having those checking.

Also im wondering is phalcon internally if working with php7 using php type hinting ? Or it's just done in zephir/c without using php zend stuff ? If not, then we could probably start using it somehow and possibly reduce extension size or not ? @andresgutierrez